lua/nvim-possession/utils.lua

```diff
@@ -25,24 +25,28 @@ M.session_files = function(file)
 	return buffers
 end
 
----return the first session whose dir corresponds to cwd
+---return the all sessions whose dir corresponds to cwd
 ---@param sessions_path string
----@return string|nil
-M.session_in_cwd = function(sessions_path)
-	local session_dir, dir_pat = "", "^cd%s*"
-	for file, type in vim.fs.dir(sessions_path) do
-		if type == "file" then
-			for line in io.lines(sessions_path .. file) do
-				if string.find(line, dir_pat) then
-					session_dir = vim.uv.fs_realpath(vim.fs.normalize((line:gsub("cd%s*", ""))))
-					if session_dir == vim.fn.getcwd() then
-						return file
-					end
-				end
-			end
-		end
-	end
-	return nil
+---@return table
+M.sessions_in_cwd = function(sessions_path)
+    local session_dir, dir_pat = "", "^cd%s*"
+    local sessions = {}
+    for file, type in vim.fs.dir(sessions_path) do
```

Comment:
This seems to be a spurious diff, or am I wrong? Could you format the file again using the `stylua.toml` provided in the repository?

Reply:
The main change to this is instead of returning the first session that matches the cwd, it returns all matching sessions.

> Could you format the file again using the stylua.toml provided in the repository?
Oh yeah of course! I was at pains trying to get the formatting to match, didn't realize there was an easy way to do so.